core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -628,7 +633,7 @@ public void accept(Path path) {
                 }
             });
         }
-        return new JarBuildItem(initJar, null, libDir, packageConfig.type);
+        return new JarBuildItem(initJar, null, libDir, packageConfig.type, null);
```

Comment:
should we bother to include a classifier for thin jar ? I don't know what the expected file output is !
Anyway the condition in BuildMojo make it attach the jar only for UberJar.

Reply:
If we were going to attach the thin jar we would need to zip it up and attach it as a .zip. I would not bother for now unless we actually have a request for this.